SPECS/libnvidia-container/libnvidia-container.spec

```diff
@@ -0,0 +1,278 @@
+%define modprobe_version 450.57
+%define _major 1
+Summary:        NVIDIA container runtime library
+Name:           libnvidia-container
+Version:        1.3.3
+Release:        2%{?dist}
+License:        BSD AND ASL2.0 AND GPLv3+ AND LGPLv3+ AND MIT AND GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/NVIDIA/libnvidia-container
+#Source0:       https://github.com/NVIDIA/%%{name}/archive/v%%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+#Source1:       https://github.com/NVIDIA/nvidia-modprobe/archive/%%{modprobe_version}.tar.gz
+Source1:        nvidia-modprobe-%{modprobe_version}.tar.gz
+Patch0:         common.mk.patch
+Patch1:         libtirpc.patch
+Patch2:         nvidia-modprobe.patch
+BuildRequires:  libseccomp-devel
+BuildRequires:  libtirpc-devel
+BuildRequires:  make
+BuildRequires:  rpcsvc-proto
+BuildRequires:  which
+
+%description
+The nvidia-container library provides an interface to configure GNU/Linux
+containers leveraging NVIDIA hardware. The implementation relies on several
+kernel subsystems and is designed to be agnostic of the container runtime.
+
+%prep
+%setup -q
+%patch0 -p1
+%patch1 -p1
+
+mkdir -p deps/src/nvidia-modprobe-%{modprobe_version}
```

Comment:
I suggest using variables to store these paths.

Reply:
changed to use a variable